ortools/linear_solver/xpress_interface.cc

```diff
@@ -1244,8 +1244,7 @@ int64_t XpressInterface::nodes() const {
 }
 
 // Transform a XPRESS basis status to an MPSolver basis status.
-static MPSolver::BasisStatus XpressToMPSolverBasisStatus(
-    int xpress_basis_status) {
+MPSolver::BasisStatus XpressToMPSolverBasisStatus(int xpress_basis_status) {
```

Comment:
this method is also declared line 489 above so if removing the static, you should change the prototype there too imho.